.github/workflows/build.yml

```diff
@@ -11,6 +11,7 @@ on:
   push:
     branches:
       - master
+      - goalsSweep
```

Comment:
Why this change?

Reply:
Ha!  Snuck in from another branch.  Needs to go!